### ISSUE TITLE
Finding issues from a private repository

### ISSUE THREAD

**Issue by lboening** [MathieuBuisson/PSGithubSearch#3]
Is it possible to find issues from a private repository? I guess I would have to log on by a function for that to work.

**Comment by MathieuBuisson** [comment]
No, none of the cmdlets in this module can access private repository, because at this time, they don't do any authentication.

**Comment by lboening** [comment]
Ok!

**Comment by lboening** [comment]
I have experimented a bit with way to do that. I am borrowing script from the famous Doug Finke.

**Comment by lboening** [comment]
Please close issue
